styleguide/source/assets/scss/02-molecules/search-result/_dr-finder-search-result-header.scss

```diff
@@ -0,0 +1,21 @@
+.dr-finder-search-result-header{
+   display: flex;
+   flex-wrap: wrap;
+   flex-direction: row;
+   gap: 0px 470px;
```

Comment:
Let's not use `gap` here, let's send this to the right side. We don't want to keep track of `gap`.